packages/babel-highlight/src/index.ts

```diff
@@ -249,25 +247,30 @@ type Options = {
  * Whether the code should be highlighted given the passed options.
  */
 export function shouldHighlight(options: Options): boolean {
-  return !!Chalk.supportsColor || options.forceColor;
+  return !!chalk.supportsColor || options.forceColor;
 }
 
-/**
- * The Chalk instance that should be used given the passed options.
- */
-export function getChalk(options: Options) {
-  return options.forceColor
-    ? new Chalk.constructor({ enabled: true, level: 1 })
-    : Chalk;
+let chalkWithForcedColor: Chalk = undefined;
+function getChalk(forceColor: boolean) {
+  if (forceColor) {
+    chalkWithForcedColor ??= new chalk.constructor({ enabled: true, level: 1 });
```

Comment:
If `forceColor` is `true`, `getChalk` will return a memoized chalk instance, previously it returned a new instance. The new behaviour aligns with the one when `forceColor` is `false`.

Reply:
Yeah I noticed it while changing this -- I think it's ok, it has the benefit of allocating less objects (+ as you noted aligning with `forceColor: false`) and no actual disadvantages.